go/libkb/pseudonym.go

```diff
@@ -30,17 +29,20 @@ type TlfPseudonymInfo struct {
 	ID      [16]byte
 	KeyGen  int
 	HmacKey [32]byte
+
+	// Ignored in requests. Set in server responses. Using untrusted data
+	// during decryption is safe because we don't rely on TLF keys for sender
+	// authenticity. (Note however that senders must not use untrusted keys, or
+	// else they'd lose all privacy.)
+	UntrustedCurrentName string
```

Comment:
Can you use a separate struct instead that embeds `TlfPseudonymInfo`? Like

```
type TlfServerPseudonymInfo struct {
  TlfPseudonymInfo
  UntrustedCurrentName string
}
```

Just so we don't have an unused field when we're dealing purely with non-server pseudonyms.